Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -77,13 +80,22 @@ namespace O3DE::ProjectManager
         FormLineEditWidget* m_originURL = nullptr;
         FormLineEditWidget* m_repositoryURL = nullptr;
 
-        TabWidget* m_tabWidget;
+        QStackedWidget* m_stackWidget = nullptr;
 
         QDialogButtonBox* m_backNextButtons = nullptr;
         QPushButton* m_backButton = nullptr;
         QPushButton* m_nextButton = nullptr;
 
+        
+        QRadioButton* m_gemTemplateSelectionTab = nullptr;
+        QRadioButton* m_gemDetailsTab = nullptr;
+        QRadioButton* m_gemCreatorDetailsTab = nullptr;
+
         GemInfo m_createGemInfo;
+
+        const int gemTemplateSelectionScreen = 0;
+        const int gemDetailsScreen = 1;
+        const int gemCreatorDetailsScreen = 2;
```

Comment:
The [code naming convention for constants](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#constant-names) use UpperCamelCase
```suggestion
        static const int GemTemplateSelectionScreen = 0;
        static const int GemDetailsScreen = 1;
        static const int GemCreatorDetailsScreen = 2;
```